package.json

```diff
@@ -22,7 +22,7 @@
     "eslint-plugin-github": "^2.0.0",
     "eslint-plugin-jest": "^22.5.1",
     "flow-bin": "^0.115.0",
-    "jest": "^24.9.0",
+    "jest": "^25.1.0",
```

Comment:
is this PR causing the \"audit tools\" step to fail?

Reply:
or was it failing before?

Reply:
looks like it's failing before. Make sure not adding new problems :)

Reply:
It was failing before, the intention was to fix it but i missed a few packages, pushing up more updates :)